docs/kickstart-docs.rst

```diff
@@ -83,6 +83,9 @@ kickstart file:
    user would be prompted during a typical installation. Once the answer
    is given, the installation will continue unattended unless it finds
    another missing item.
+-  Commands mentioned in the ``method`` proxy command are required for
```

Comment:
It looks like all of these commands are required.

Reply:
Huh, true..